api/app/src/routes/medical/document.ts

```diff
@@ -109,10 +111,12 @@ router.get(
     const cxId = getCxIdOrFail(req);
     const fileName = getFromQueryOrFail("fileName", req);
     const fileHasCxId = fileName.includes(cxId);
+    const type = getFrom("query").optional("conversionType", req);
+    const conversionType = docConversionTypeSchema.parse(type);
```

Comment:
Shouldn't we only parse if present? I seems this makes the `conversionType` param required